quizzes/08_spread.js

```diff
@@ -0,0 +1,83 @@
+console.log(
+  // getMax([7, 8, 5, 9, 20, 7])
```

Comment:
Thank you very much for your willingness to iterate with me on this stuff. I hope you're having a good time and learning stuff!